### DISCUSSION TITLE
Remuneration for select PRs via OnlyDust

### DISCUSSION THREAD

**Discussion by stoobie** [starknet-io/starknet-docs#1091] [Announcements]
We are starting an initiative in which the Starknet Foundation is funding select PRs via OnlyDust. The relevant PRs are labeled with the golden **Remuneration provided** label. 
![image](https://github.com/starknet-io/starknet-docs/assets/39279277/b390e4d5-7770-437c-b68e-0c0d12066f1a)

Please be aware that for now, in order to view a preview of how the PR will actually appear on the site, you need to generate the site on your local machine and view it using a local web server. The contribution guidelines now provide instructions on how to generate the site locally.

**Comment by stoobie** [comment]
I previously added a few folks who were actively contributing as collaborators. I did so as a temporary workaround to enable those folks to generate previews, because, as noted in issue #1043, site previews for PRs are not being generated when a PR originates from a fork.

Following up on the above, I am removing a few folks from being official collaborators. This is not something negative, rather I am doing it to enable those folks to participate in PRs for which remuneration is being provided.

@xiaolou86 @hardiktheprogrammer FYI.

**Comment by xiaolou86** [comment]
It's a big event!  @stoobie thank you very much for your introduction and giving us the chance!
